src/mimeparser.rs

```diff
@@ -4088,4 +4147,24 @@ Content-Type: text/plain; charset=utf-8
             "alice@example.org"
         );
     }
+
+    #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+    async fn test_protect_autocrypt() -> Result<()> {
+        let mut tcm = TestContextManager::new();
+        let alice = &tcm.alice().await;
+        let bob = &tcm.bob().await;
+
+        alice
+            .set_config_bool(Config::ProtectAutocrypt, true)
+            .await?;
+        bob.set_config_bool(Config::ProtectAutocrypt, true).await?;
```

Comment:
Seems `rustfmt` has weird default setting, 67 chars are ok, but 69 are already wrapped. I think at least 80 never need wrapping

Reply:
It's https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=#chain_width (60 w/o indentation). Also see https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=#use_small_heuristics for other defaults. I'd suggest to set the limit for all statements to 80 chars, that was always fine as far as i remember

Reply:
Yes, this isn't an option. Unfortunately there's no option to preserve already existing line breaks :/